vllm_ascend/ops/fused_moe.py

```diff
@@ -50,6 +51,14 @@ def fused_experts_with_mc2(
 ) -> torch.Tensor:
     global_bs = 0
     moe_expert_num = len(expert_map)
+
+    rank = torch.distributed.get_rank()
+    if VLLM_ENABLE_FIX_ROUTE:
```

Comment:
Please make sure `VLLM_ENABLE_FIX_ROUTE` is still needed or not after https://github.com/vllm-project/vllm-ascend/commit/afc4c0cd035ee22d51950d24b7f642abcaa0b176